src/main/scala/com/codacy/analysis/cli/utils/EitherOps.scala

```diff
@@ -0,0 +1,34 @@
+package com.codacy.analysis.cli.utils
+
+import scala.util.{Failure, Success, Try}
+
+object EitherOps {
+
+  def sequenceWithFixedLeftUnit[A](left: A)(eitherIterable: Seq[Either[A, Unit]]): Either[A, Unit] = {
+    eitherIterable
+      .foldLeft[Either[A, Unit]](Right(())) { (acc, either) =>
+        acc.flatMap(_ => either)
+      }
+      .left
+      .map(_ => left)
+  }
+
+  def sequenceWithFixedLeft[A, B](left: A)(eitherIterable: Set[Either[A, B]]): Either[A, Set[B]] = {
+    eitherIterable
+      .foldLeft[Either[A, Set[B]]](Right(Set.empty)) { (acc, either) =>
+        acc.flatMap { bSeq =>
+          either.map(b => bSeq ++ Set(b))
+        }
+      }
+      .left
+      .map(_ => left)
+  }
+
+  def fromTry[A](tryValue: Try[A]): Either[String, A] = {
```

Comment:
This seems redundant with TryOps.toRight
(if you don't want to change the existing one we can have another one that has Throwable => L as the argument)